queries.sql

```diff
@@ -28,4 +28,64 @@ WHERE name != 'Gabumon';
 
 SELECT *
 FROM animals
-WHERE weight_kg BETWEEN 10.4 AND 17.3;
\ No newline at end of file
+WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+-----
+BEGIN;
+    UPDATE animals
+SET species = 'unspecified';
+    ROLLBACK;
+END;
```

Comment:
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_update_tables.md#:~:text=to%20unspecified.-,Verify%20that%20change%20was%20made.%20Then%20roll%20back%20the%20change%20and%20verify%20that%20the%20species%20columns%20went%20back%20to%20the%20state%20before%20the%20transaction.,-Inside%20a%20transaction), Please verify that change was made and verify that the species columns went back after the rollback.

Hint: 
```sql
   BEGIN; 
   UPDATE animals
   SET species = 'unspecified'; 
   SELECT species from animals; -- verify that change was made
   ROLLBACK;
   SELECT species from animals; -- verify that change was undone
```

Reply:
As requested by the previous reviewer, kindly verify whether changes were made and verify specifies column was set back to before setting it to `unspecified`. It is a part of the project requirement.